### PR TITLE
Arcanization

### PR DESCRIPTION
**Pull request by nical**

### CODE REVIEW

File: _posts/2023-11-24-arcanization.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The next release featuring this work will be `0.19.0` which we expect to publish around January 17th. We made sure to merge the changes early in the release cycle to give ourselves as much time as we can to catch potential regressions.
+
+This is an absolutely massive change and while we have and are testing as best we can, we do need help from everyone else. Please try updating your project to the latest wgpu and running it. Please report any issues you find!
+
+# What's next?
+
+## Lifting `RenderPass<'a>` lifetime restrictions
+
+If you have used `wgpu`, there is decent chance that you have had to work around the restrictions imposed by the `'rpass` lifetime in a lot of `RenderPass`'s methods, such as `set_bind_group`, `set_pipeline`, and, `set_vertex_buffer`. The recent changes give us the opportunity to store `Arc`s where `&'a` references were previously needed which should let us remove these lifetime restrictions. A [pull request](github.com/gfx-rs/wgpu/pull/4768) is already open to address this.
```
Comment (cwfitzgerald) [inline _posts/2023-11-24-arcanization.md:49]: We shouldn't mention this PR, it doesn't actually fix the problem, just removes the lifetimes.

**Review by cwfitzgerald — CHANGES_REQUESTED** [review]

**Review by cwfitzgerald — APPROVED** [review]
Nice!

**Comment by a1phyr** [comment]
Hi ! Thanks for this blog post !

The images links are though.

**Comment by cwfitzgerald** [comment]
They should be fixed now

**Pull request merged**